### PR TITLE
Feature/retrieve aspect data frontend

### PR DESCRIPTION
**Pull request by saudkhan116**
# Why we create this PR?
 
Added a method `getAspectData` to retrieve aspect based on verification details.
case 1: the data is retrieved without having verified it
case 2: the data is retrieved with verification details.
case 3: if aspect data is null, print error message 
 
# What we want to achieve with this PR?
 
To print verification related data in frontend

# What is new?
 
## Updated
- Updated the way of data retrieval in frontend based on verification details.

### CODE REVIEW

**Review by matbmoser — APPROVED** [review]

File: dpp-frontend/src/views/PassportView.vue [hunk 1]
```diff
@@ -28,19 +28,11 @@
       <template v-if="!data">
         <span class="header-title">{{ $t("passportView.dpp") }}</span>
       </template>
-      <template
-        v-else-if="
-          data.semanticId ===
-          'urn:bamm:io.catenax.battery.battery_pass:3.0.1#BatteryPass'
-        "
-      >
+      <template v-else-if="data.semanticId === 'urn:bamm:io.catenax.battery.battery_pass:6.0.0#BatteryPass'">
```
Comment (matbmoser) [inline dpp-frontend/src/views/PassportView.vue:31]: This is wrong

File: dpp-frontend/src/views/PassportView.vue [hunk 2]
```diff
@@ -855,19 +697,36 @@ export default {
...
             }
+
+            // process aspect data based on vc or not vc is available
+            // case 1: if verifiable credntials vc is not present in data (vc=false),
+            // then only passport is shown without verification details
+            if (!this.data["verification"]["vc"]) aspect = jsonUtil.get("data.aspect", this.data);
+
+            // case 2: if verifiable credntials vc is present in data (vc=true),
+            // then passport with verification details is shown
+            if (this.data["verification"]["vc"]) aspect = passportUtil.getAspectData(this.data);
```
Comment (matbmoser) [inline dpp-frontend/src/views/PassportView.vue:715]: If there is no "verification" in the data there is an exception no?

Comment (saudkhan116) [reply dpp-frontend/src/views/PassportView.vue]: fixed

**Comment by matbmoser** [comment]
Can we solve this bug #274 in this PR

**Review by matbmoser — APPROVED** [review]
Approved nice job

**Pull request merged**